engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/exception/WithFlinkEspExceptionHandler.scala

```diff
@@ -0,0 +1,27 @@
+package pl.touk.nussknacker.engine.flink.api.exception
+
+import org.apache.flink.api.common.functions.{RichFunction, RuntimeContext}
+import org.apache.flink.configuration.Configuration
+
+/**
+  * Helper for using exception handler.
+  *
+  * Be aware that super.open and super.close are not called.
+  */
+trait WithFlinkEspExceptionHandler {
+  self: RichFunction =>
```

Comment:
I wonder if it's a good practice to do something like this?
If you implement multiple traits like this in your RichFunction you will have valid scala code, but actually methods from only one trait will be executed.
So this trait cannot be used with e.g `LazyParameterInterpreterFunction`

Reply:
But do we have better way to do this?

Reply:
That's why I added a comment that open and close from base class are never called.

Reply:
If there is no better way, I guess I will have to live with it :)